ui/src/error.ts

```diff
@@ -5,12 +5,20 @@
 // LICENSE file.
 
 import * as svelteStore from "svelte/store";
-import * as lodash from "lodash";
+import lodash from "lodash";
 
-import * as config from "ui/src/config";
-import * as notification from "./notification";
+import * as bacon from "./bacon";
 import * as ipc from "./ipc";
 
+// We use an event bus to break up the dependency cycle between the `error`,
+// `notification` and `browserStore` modules. This also makes the mocks in the
+// unit tests simpler.
+const errorBus = new bacon.Bus<Error>();
+export const errorBusOutput = errorBus.toEventStream();
+export function showErrorNotification(error: Error): void {
```

Comment:
We can align the names a bit better. No need to mention errors explicitly since this is the `error` module.

```suggestion
const notificationsBus = new bacon.Bus<Error>();
export const notifications: bacon.EventStream<Error> = errorBus.toEventStream();
export function showNotification(error: Error): void {
```

Reply:
3aeb9693966575f9184ff7e2ef291afd7edb4090